resources/user-guides/attach-data.md

```diff
@@ -0,0 +1,113 @@
+# Using the Attach Data Rules Action
+
+The Attach Data rule action was added in the `Mobile Core v2.1.8` Launch extension. Attach Data is powerful and complex, and is recommended for users with advanced use cases.
```

Comment:
- Start the 2nd sentence as "This rule..."
- Please remove the comma between "complex" and "and"